.github/actions/assign-release-task/action.yml

```diff
@@ -0,0 +1,67 @@
+name: 'Assign Android Release Task in Asana'
+description: 'Assigns the latest Asana Release task to the user who runs the workflow'
+inputs:
+  task_name:
+    description: 'The name of the task to search for'
+    required: true
+  asana_token:
+    description: 'Asana Personal Access Token'
+    required: true
+  project_gid:
+    description: 'Asana Project GID to search within'
+    required: true
+  username:
+    description: 'The Github username to search for'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - name: Find task in Asana and assign it to owner
+      shell: bash
+      run: |
+        task_name="${{ inputs.task_name }}"
+        asana_token="${{ inputs.asana_token }}"
+        project_gid="${{ inputs.project_gid }}"
+        username="${{ inputs.username }}"
+
+        # Make the API request to get tasks from the specified project
```

Comment:
`aarb` already knows the release task ID so we could probably work out a way so that it returns it directly, and save the extra search on the Asana API based on the task name. 

but could be filed away as a future improvement assuming the searching approach will work fine.

Reply:
Yes, that’s a good improvement. For now, I’d prefer to keep it as is and file an improvement to aarb at a later stage.